Modules/Setup

```diff
@@ -137,6 +137,7 @@ PYTHONPATH=$(COREPYTHONPATH)
 #_datetime _datetimemodule.c
 #_decimal _decimal/_decimal.c
 #_heapq _heapqmodule.c
+_fnmatch _fnmatchmodule.c
```

Comment:
I don't really know if I need to do more here... There is no tutorial on how to add a new C extension to the core library so if someone can help me for that :(

Reply:
(And that's the reason why there are failures... I think I committed thr wrong file)